src/datatypesconditions/Question7.java

```diff
@@ -0,0 +1,34 @@
+package datatypesconditions;
+
+import java.util.Scanner;
+
+/**
+ * Created by Vane Gharakhanian on Sep, 2020
+ */
+public class Question7 {
+// Take three numbers from the user and print the greatest number.
+
+    public static void main(String[] args) {
+        int a = 0, b = 0, c = 0;
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number:");
+        a = scanner.nextInt();
+        System.out.println("Enter a number:");
+        b = scanner.nextInt();
+        System.out.println("Enter a number:");
+        c = scanner.nextInt();
+
```

Comment:
simply do it: 

if (a > b && a > c) {
            System.out.println("biggest number is: " + a);
  } else {
            System.out.println("biggest number is: " + Math.max(b, c));
 }